pkg/utils/strings.go

```diff
@@ -1,10 +0,0 @@
-package utils
```

Comment:
What is the relation of this change to config maps?

Reply:
Clayton wanted to remove the package in a comment to previous review.

Reply:
But this could be another PR as it's not related to this change.